common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1387,6 +1387,12 @@
     ],
     "sqlState" : "42623"
   },
+  "COLUMN_DEFAULT_VALUE_IS_NOT_FOLDABLE_OR_RESOLVED" : {
+    "message" : [
+      "The existence default value must be a simple SQL string that is resolved and foldable, but column <colName> has default value: (<defaultValue>)"
```

Comment:
Not sure what `existence default value` means here, I think it is more clear if it is just `default value`.

Reply:
Yeah, there are existence and current default values.

Reply:
but for end users they only know column default value?

Reply:
Actually yes, they don't know